ui/src/views/admin/users/UsersView.vue

```diff
@@ -471,40 +460,40 @@ export default {
     }
   },
   created() {
-    // get user api
-    // add certain columns to user data
+    this.getUsersApi();
     this.processUserData();
   },
   mounted() {},
   methods: {
-    onDeleteConfirmation() {
-      // call delete api
-      // delete
-      // refresh table
-      // hide modal
-      this.isDeleteItemModalShown = false;
-    },
     onClickItemDelete(item) {
       this.selectedUser = { ...item };
       this.isDeleteItemModalShown = true;
     },
+    onDeleteConfirmation() {
+      const apiData = { email: this.selectedUser.email };
+      adminDeleteUser(apiData).then((response) => {
+        this.$notify.toast(response["message"]);
+        this.isDeleteItemModalShown = false;
+        window.location.reload();
```

Comment:
Why reload the pages? I think its better to refetch the api and refresh the component instead in case its not loaded properly.

Reply:
I thought reloading the page is the quickest programming way to reload new data and ensure FE and BE are synced properly. I remember in the past I had some annoying issues where the datatable didn't want to re-render with new data, and if it happens I am afraid it will take hours to debug (and I've spent way too many hours debugging things in the past few days, when the task of connecting apis should've been finished). But I can try refetch and refresh the component.

Reply:
Okay, so I have changed it so that it doesn't refresh the page, instead it will refetch api. I have tested it and it seems to be working (fortunately XD).

Reply:
It should be working, whats great about js framework is that they are reactive XD  there might be some issues when the components are nested deep but here we only got one component to be updated and its the direct children, so it should be fine